components/display/ili9341.rst

```diff
@@ -0,0 +1,128 @@
+ILI9341 TFT LCD
+===============
+
+.. seo::
+    :description: Instructions for setting up ILI9341 TFT LCD display drivers.
+    :image: ili9341-full.jpg
+
+.. _ili9341:
+
+Usage
+-----
+
+The ``ili9341`` display platform allows you to use
+ILI9341 (`datasheet <https://cdn-shop.adafruit.com/datasheets/ILI9341.pdf>`__,
+`Aliexpress <https://s.click.aliexpress.com/e/_dTKH6Mt>`__)
+displays with ESPHome. As this is a somewhat higher resolution display and may require pins
+beyond the typical SPI connections, it is better suited for use with the ESP32.
+
+.. figure:: images/ili9341-full.jpg
+    :align: center
+    :width: 75.0%
+
+    ILI9341 display
+
+
+.. code-block:: yaml
+
+    # Example minimal configuration entry
+    display:
+      - platform: ili9341
+        model: TFT 2.4
+        cs_pin: 14
+        dc_pin: 27
+        led_pin: 32  ### see note below ###
+        reset_pin: 33
+        rotation: 0
+        
+        lambda: |-
+          it.fill(ili9341::COLOR_BLACK);
```

Comment:
Does this works? Should it work?

Reply:
I will confirm...could be that I missed part of it. I'll check and fix as necessary.

Reply:
A find/replace got the better of me. This is fixed and confirmed to work.